app/src/main/java/org/fossasia/openevent/app/data/AbstractObservable.java

```diff
@@ -18,15 +19,19 @@
 public final class AbstractObservable {
 
     private final ConnectionStatus connectionStatus;
+    private final RateLimiter<String> rateLimiter = new RateLimiter<>(Duration.ofMinutes(10));
```

Comment:
Also, about this rate limiter, will it not disallow people to pull to refresh?

Reply:
No it won't, An API call is made in two cases: 
1. `reload` is true.
2. An API call wasn't made in the last specified duration(10 minutes in my case).

``` Java
if (reload || abstractObservable.rateLimiter.shouldFetch(rateLimiterKey)) {
                    return networkObservable
                        .doOnNext(item -> Timber.d("Loaded %s From Network on Thread %s",
                            item.getClass(), Thread.currentThread().getName()));
                }
```

Reply:
OK, thanks

Reply:
Please change the builder constructor and fix the tests